src/app/tables/artifacts-table/artifacts-table.component.ts

```diff
@@ -86,37 +91,37 @@ export class ArtifactsTableComponent implements OnChanges {
     let columns: Array<string>;
     switch (resource.toLowerCase()) {
       case ('bugzillabug'):
-        columns = ['id', 'modified_time', 'assignee', 'reporter', 'short_description', 'status'];
+        columns = ['id', 'modified_time', 'assignee', 'reporter', 'short_description', 'status', 'link'];
         break;
       case ('distgitcommit'):
-        columns = ['author', 'hash', 'commit_date', 'log_message'];
+        columns = ['author', 'hash', 'commit_date', 'log_message', 'link'];
         break;
       case ('kojibuild'):
       case ('containerkojibuild'):
       case ('modulekojibuild'):
-        columns = ['id', 'completion_time', 'name', 'owner', 'release', 'version'];
+        columns = ['id', 'completion_time', 'name', 'owner', 'release', 'version', 'link'];
         break;
       case ('containeradvisory'):
       case ('advisory'):
-        columns = ['advisory_name', 'update_date', 'assigned_to', 'id', 'security_impact', 'state', 'synopsis'];
+        columns = ['advisory_name', 'update_date', 'assigned_to', 'id', 'security_impact', 'state', 'synopsis', 'link'];
         break;
       case ('freshmakerevent'):
-        columns = ['id', 'state_name', 'state_reason', 'successful_koji_builds', 'time_created'];
+        columns = ['id', 'state_name', 'state_reason', 'successful_koji_builds', 'time_created', 'link'];
         break;
       case ('distgitrepo'):
-        columns = ['commits', 'name', 'namespace'];
+        columns = ['commits', 'name', 'namespace', 'link'];
         break;
       case ('distgitbranch'):
-        columns = ['repos', 'repo_name', 'repo_namespace'];
+        columns = ['repos', 'repo_name', 'repo_namespace', 'link'];
         break;
       case ('kojitag'):
-        columns = ['builds', 'id', 'name'];
+        columns = ['builds', 'id', 'name', 'link'];
         break;
       case ('freshmakerbuild'):
-        columns = ['id', 'name', 'original_nvr', 'rebuilt_nvr', 'state_name', 'state_reason'];
+        columns = ['id', 'name', 'original_nvr', 'rebuilt_nvr', 'state_name', 'state_reason', 'link'];
         break;
       default:
-        columns = ['id'];
+        columns = ['id', 'link'];
```

Comment:
Optional: Instead of adding `'link'` to all of the columns individually, you can just do `columns.push('link')` after the switch statement.